src/crypto/api.ts

```diff
@@ -100,17 +100,3 @@ export interface ICreateSecretStorageOpts {
      */
     getKeyBackupPassphrase?: () => Promise<Uint8Array>;
 }
-
-export interface IImportOpts {
-    stage: string; // TODO: Enum
-    successes: number;
-    failures: number;
-    total: number;
-}
-
-export interface IImportRoomKeysOpts {
-    /** called with an object that has a "stage" param */
-    progressCallback?: (stage: IImportOpts) => void;
-    untrusted?: boolean;
-    source?: string; // TODO: Enum
-}
```

Comment:
applications might be depending on the existence of these interfaces, so we need to re-export `ImportRoomKeysOpts` and `ImportOpts` for backwards compatibility (see the existing "re-exports for backwards compatibility" at the top of the file).

Reply:
re-exported in api.ts
 ImportRoomKeyProgressData as IImportOpts,
 ImportRoomKeysOpts as IImportRoomKeysOpts,